src/main/kotlin/com/ajaxproject/telegrambot/bot/repository/FinanceRepository.kt

```diff
@@ -0,0 +1,14 @@
+package com.ajaxproject.telegrambot.bot.repository
+
+import com.ajaxproject.telegrambot.bot.enums.Finance
+import com.ajaxproject.telegrambot.bot.models.MongoFinance
+import org.bson.types.ObjectId
+
+interface FinanceRepository {
+
+    fun findByUserId(userId: Long, financeType: Finance): List<MongoFinance>?
```

Comment:
Please, avoid null-related issues. Maybe you mean that?
```suggestion
    fun findByUserId(userId: Long, financeType: Finance): List<MongoFinance?>
```